docs/en/guide/deploy.md

```diff
@@ -166,13 +186,19 @@ Don't enable options like _Auto Minify_ for HTML code. It will remove comments f
          - name: Setup Pages
            uses: actions/configure-pages@v4
```

Comment:
With vitepress@1.6.3 and custom URLs I get the following error when using `actions/configure-pages@v4`

```
Error: Get Pages site failed. Please verify that the repository has Pages enabled and configured to build using GitHub Actions, or consider exploring the `enablement` parameter for this action.
```

upgrading to  `actions/configure-pages@v5` resolved this